bincrafters_conventions/bincrafters_conventions.py

```diff
@@ -355,21 +397,30 @@ def _clone_project(self, github_url):
         project = github_url[(github_url.rfind('/') + 1):]
         project_path = os.path.join(temp_dir, project)
         repo = git.Repo.clone_from(github_url, project_path)
-        self._logger.info("Clone project {} to {}".format(github_url, project_path))
+        self.output_remote_update("Clone project {} to {}".format(github_url, project_path))
         return repo, project_path
 
     def _list_user_projects(self, user):
-        """ List all projects from Github public account
+        """ List all projects from GitHub public account
 
         :param user: User name
         """
         projects = []
-        repos_url = 'https://api.github.com/users/{}/repos'.format(user)
-        response = requests.get(repos_url)
-        if not response.ok:
-            raise Exception("Could not retrieve {}".format(repos_url))
-        for project in response.json():
-            projects.append(project["full_name"])
+        pages = (1, 2)
+        for page in pages:
+            repos_url = 'https://api.github.com/users/{}/repos?sort=updated&direction=asc&per_page=100&page={}'\
+                .format(user, page)
+            response = requests.get(repos_url)
+            if page == 1 and not response.ok:
+                raise Exception("Could not retrieve {}".format(repos_url))
+            for project in response.json():
+                if project["name"].startswith("conan-") \
+                        and not project["name"].startswith("conan-boost") \
+                        and not project["archived"] \
+                        and not project["fork"] \
```

Comment:
we have some forked projects, not sure if we need to skip them

Reply:
For the permanent forks we should write GitHub support an email to remove the fork status.

However, we also have sometimes temporary forks and I guess they shouldn't be updated by this script

Reply:
okay, thank you for the clarification